image_width: 960
image_height: 720
camera_name: limelight
camera_matrix:
  rows: 3
  cols: 3
  
  data: [
    772.53876202, 0., 479.132337442,
    0., 769.052151477, 359.143001808,
    0., 0., 1.0
  ]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [2.9684613693070039e-01, -1.4380252254747885e+00,-2.2098421479494509e-03, -3.3894563533907176e-03, 2.5344430354806740e+00]
rectification_matrix:
  rows: 3
  cols: 3
  data: [
    1.000000, 0.000000, 0.000000,
    0.000000, 1.000000, 0.000000,
    0.000000, 0.000000, 1.000000
  ]
projection_matrix:
  rows: 3
  cols: 4
  data: [
    772.53876202, 0., 479.132337442, 0.0,
    0., 769.052151477, 359.143001808, 0.0,
    0., 0., 1.0, 0.0
  ]
